boilerplate/fullstack/next-app-dir-multitenancy/app/auth/[[...path]]/page.tsx

```diff
@@ -11,11 +11,6 @@ import { ChangeTenantsButton } from "../../../app/components/changeTenantsButton
 import { TenantSelector } from "../../../app/components/TenantSelector";
 
 export default function Auth() {
-    if (typeof window === "undefined") {
-        return null;
```

Comment:
this project also has `SessionAuthForNextJS` which would need to change just like the non multi tenancy version?

Reply:
Yup, I missed that. I remembered checking but it looks like I stopped checking when the auth page was OK.